dev: add admin repair table statement (#1795)
pingcap/docs: dev/reference/configuration/tidb-server/configuration-file.md

- Determines whether to treat the `utf8` character set in old tables as `utf8mb4`.
- Default value: `true`

### `repair-mode`

- Determines whether to enable the untrusted repair mode. When the `repair-mode` is set to `true`, bad tables in the `repair-table-list` cannot be loaded.
- Default value: `false`

> - The syntax for the `repair` is not supported by default, and all table information is loaded when TiDB is started.

Reviewer: ```suggestion
- The `repair` syntax is not supported by default. This means that all tables are loaded when TiDB is started.
```

### `repair-table-list`

> - Lists all bad tables that need to be fixed in instances in the case that the `repair-mode` is set to `true`. An example for the list is: ["db.table1","db.table2"...].

Author: Thanks. The file is updated.

> - Lists all bad tables that need to be repaired in instances in the case that the `repair-mode` is set to `true`. An example for the list is: ["db.table1","db.table2"...].

Reviewer: ```suggestion
- `repair-table-list` is only valid when [`repair-mode`](#repair-mode) is set to `true`. `repair-table-list` is a list of bad tables that need to be repaired in an instance. An example of the list is: ["db.table1","db.table2"...].
```

- Default value: []

> - With the default setting, the list is empty, which means that there is no information about bad tables that need to be fixed.

Reviewer 2: ```suggestion
- With the default setting, the list is empty, which means that there is no information about bad tables that need to be repaired.
```

> - With the default setting, the list is empty, which means that there is no information about bad tables that need to be repaired.

Reviewer: ```suggestion
- The list is empty by default. This means that there are no bad tables that need to be repaired.
```
